dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -164,16 +164,87 @@ dashboard.showProjectHeader = function () {
 
   $('.project_share').click(function () {
     dashboard.saveProject(function () {
-      var dialog = new Dialog({body: '<div id="project-share" class="modal-content">'
+      var dialog = new Dialog({body: '<img class="modal-image" src="' + appOptions.skin.staticAvatar + '">'
+          + '<div id="project-share" class="modal-content">'
           + '<p class="dialog-title"><%= I18n.t('project.share_title') %></p>'
           + '<p><%= I18n.t('project.share_copy_link') %></p>'
-          + '<div><input type="text" id="sharing-input" value="' + location.href.split(/\/edit$/)[0] + '" style="cursor: copy; width: 400px;" readonly></div>'
-          + '<button id="continue-button"><%= I18n.t('project.ok') %></button></div>'
+          // TODO: de-dup with apps code once JS common-core work is done.
```

Comment:
is it worth making this a partial if only so we don't have to deal with mentally parsing all the nested quotes?